election-api-scala/tasks.md

```diff
@@ -16,16 +16,10 @@ The scoreboard should show:
 - The seats for each party
 - The overall winner (i.e. the party with 325 or more seats) if there is one
 
-There is a case class [Scoreboard](app/model/PartyResult.scala) ready to be extended and an
+There is a case class [Scoreboard](app/model/PartyResult.scala) which you should alter to show the above results and an
 endpoint controller to put your implementation: [ResultsController](app/controllers/ResultsController.scala).   You will also need to complete 
 the [tests](test/controllers/ResultsControllerSpec.scala) as part of your implementation.
 
 Bonus information for the scoreboard:
 - The total votes for each party
 - The total share of the vote for each party. So the percentage of votes for each party.
-
-
-### Possible other implementations
-
-- Absolute majority required. Someone needs 50% + 1 votes or a run off is triggered (check the data that's probably all constituencies)
-- Allocate the seats from the total declarations based on % of vote share
```

Comment:
Agree, the whole "other implementations" section looking at other voting systems is a bad idea (but! for fairness/consistency we should edit tasks.md for other languages)